src/ape/contracts/base.py

```diff
@@ -621,6 +621,11 @@ def poll_logs(
         required_confirmations = (
             required_confirmations or self.provider.network.required_confirmations
         )
+        
+        height = max(self.chain_manager.blocks.height - required_confirmations, 0)
+        if start_block and height > 0 and start_block < height:
```

Comment:
would this be better as `start_block is not None` because of `0`?

Reply:
per conversations with @fubuloubu in DM, we did it this way because if start_block == 0 the range call will fail